e2e/Makefile

```
@@ -193,7 +201,11 @@ test: create-cluster
 	$(KUBECTL) apply -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/$(EXTERNAL_SNAPSHOTTER_VERSION)/client/config/crd/snapshot.storage.k8s.io_volumesnapshots.yaml
 	$(KUBECTL) apply -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/$(EXTERNAL_SNAPSHOTTER_VERSION)/deploy/kubernetes/snapshot-controller/rbac-snapshot-controller.yaml
 	$(KUBECTL) apply -f https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/$(EXTERNAL_SNAPSHOTTER_VERSION)/deploy/kubernetes/snapshot-controller/setup-snapshot-controller.yaml
-	$(KUBECTL) apply -f manifests/common/
+	if [ "$(USE_LEGACY)" = "true" ]; then \
+		$(KUBECTL) apply -f manifests/provisioner_legacy.yaml; \
+	else \
+		$(KUBECTL) apply -f manifests/provisioner.yaml; \
+	fi
```

```suggestion
ifeq ($(USE_LEGACY),true)
	$(KUBECTL) apply -f manifests/provisioner_legacy.yaml
else
	$(KUBECTL) apply -f manifests/provisioner.yaml
endif
```

Or it may be nice to define a variable and switch its value like `SCHEDULER_CONFIG` and `HELM_VALUES_FILE`.

---

> Or it may be nice to define a variable and switch its value like SCHEDULER_CONFIG and HELM_VALUES_FILE.

I fixed it in this way. Thanks.